numberformat/diff.emu

```diff
@@ -1176,15 +1176,15 @@
         1. If _x_ is ~not-a-number~ or _y_ is ~not-a-number~, throw a *RangeError* exception.
         1. If _x_ is a mathematical value, then
           1. If _y_ is a mathematical value and _y_ &lt; _x_, throw a *RangeError* exception.
-          1. Else if _y_ is ~negative-infinity~, throw a *RangeError* exception.
-          1. Else if _y_ is ~negative-zero~ and _x_ &ge; 0, throw a *RangeError* exception.
-        1. Else if _x_ is ~positive-infinity~, then
+          1. If _y_ is ~negative-infinity~, throw a *RangeError* exception.
+          1. If _y_ is ~negative-zero~ and _x_ &ge; 0, throw a *RangeError* exception.
+        1. If _x_ is ~positive-infinity~, then
           1. If _y_ is a mathematical value, throw a *RangeError* exception.
-          1. Else if _y_ is ~negative-infinity~, throw a *RangeError* exception.
-          1. Else if _y_ is ~negative-zero~, throw a *RangeError* exception.
-        1. Else if _x_ is ~negative-zero~, then
```

Comment:
I do not think you should remove this Else here because not all the condictions inside the previous block will throw or return so keep this inside else save unnecessary check.